### PR TITLE
add all links to homepage for pension

### PR DESCRIPTION
**Pull request by accbjt**
## Description
Add Pensions: Homepage links

## Testing done
Manual Testing

## Screenshots
Can't test with screenshots

## Acceptance criteria
- [ ] all links on Pension homepage should match what is on the spreadsheet.

https://docs.google.com/spreadsheets/d/1SsA5RVkERpMF8DE4Oal7fy4dK617JkdJgMQ2zrRdJXc/edit#gid=1110696968

## Definition of done
- [ ] Events are logged appropriately
- [ ] Documentation has been updated, if applicable
- [ ] A link has been provided to the originating GitHub issue (or connected to it via ZenHub)
- [ ] No sensitive information (i.e. PII/credentials/internal URLs/etc.) is captured in logging, hardcoded, or specs

### CODE REVIEW

**Comment by accbjt** [comment]
@DanielleSoCompany @mnorthuis @bethpotts On the Pension homepage there was a missing link in the spreadsheet. The link for `File an Appeal` is not there. I am not sure if we have a url for this, but I wanted to make sure you were aware of it. Please let me know before this gets approved and merged.

File: va-gov/pages/pension/index.md [hunk 1]
```diff
@@ -14,19 +14,19 @@ promo:
...
+    - url: "/housing-assistance/disability-housing-grants/"
       title: <b>Disability Housing Grants for Veterans</b>
       description: Learn how to apply for a housing grant as a Veteran or Servicemember with a service-connected disability.
-    - url: "#"
+    - url: "/careers-employment/vocational-rehabilitation/"
       title: <b>Vocational Rehabilitation and Employment</b>
       description: Find out if you're eligible for help exploring employment options, any training you may need, and other voc rehab services.
     - url: "#"
       title: <b>Family and Caregiver Health Benefits</b>
       description: See if you qualify for VA medical benefits as a spouse, surviving spouse, dependent child, or caregiver.
```
Comment (RobAWilkinson) [inline va-gov/pages/pension/index.md]: I think this link is gone according to the spreadsheet

File: va-gov/pages/pension/index.md [hunk 2]
```diff
@@ -152,12 +152,12 @@ VA pension benefits provide tax-free monthly payments to wartime Veterans with l
   <h2>More Information and Resources</h2>
```
Comment (RobAWilkinson) [inline va-gov/pages/pension/index.md]: The links for "Check Your VA Claim or Appeal Status", "View Your VA Payment History", and "Change Your VA Direct Deposit and Contact Information" are now `/claim-or-appeal-status/`, `/va-payment-history/, and `/change-direct-deposit-and-contact-information/` according to the spreadsheet.

File: va-gov/pages/pension/index.md (hunk 1 above)
Comment (RobAWilkinson) [reply va-gov/pages/pension/index.md]: "Family and Caregiver Health Benefits"

File: va-gov/pages/pension/index.md (hunk 2 above)
Comment (RobAWilkinson) [reply va-gov/pages/pension/index.md]: I only caught it cuz I just caught it on myself haha

**Comment by DanielleSoCompany** [comment]
Hi @accbjt. I think File an Appeal should go to /disability/file-an-appeal/. But I'd like to ask @mnorthuis to confirm just to make sure. Thanks!

**Comment by mnorthuis** [comment]
File an Appeal was removed back in August. It should not be on the Hub page.

**Comment by DanielleSoCompany** [comment]
Hmm, okay. @accbjt Do you want to remove as part of this PR, or do you want me to open a new PR for removing this from hub page?

**Comment by accbjt** [comment]
@DanielleSoCompany I'll remove it as a part of this PR

**Review by RobAWilkinson — APPROVED** [review]

**Pull request merged**